pre_commit/languages/docker.py

```diff
@@ -15,6 +18,32 @@
 healthy = helpers.basic_healthy
 
 
+def is_in_docker() -> bool:
+    try:
+        with open('/proc/1/cgroup') as f:
+            return 'docker' in f.read()
```

Comment:
it would be slightly better to read this as binary `, 'rb'` and then do `b'docker'` in ...

Reply:
done

Reply:
done